src/goto-programs/goto_inline.cpp

```diff
@@ -198,7 +198,7 @@ void goto_partial_inline(
       if(goto_function.is_inlined() ||
          goto_program.instructions.size()<=smallfunc_limit)
       {
-        assert(i_it->is_function_call());
+        INVARIANT(i_it->is_function_call(), "is a call");
```

Comment:
If possible could the message include a short explanation of why this should be impossible?